sklearn/preprocessing/data.py

```diff
@@ -2873,3 +2875,32 @@ def inverse_transform(self, X):
                     X_tr[mask, idx] = None
 
         return X_tr
+
+    def get_feature_names(self, input_features=None):
+        """Return feature names for output features
+
+        Parameters
+        ----------
+        input_features : list of string, length n_features, optional
+            String names for input features if available. By default,
+            "x0", "x1", ... "xn_features" is used.
+
+        Returns
+        -------
+        output_feature_names : list of string, length n_output_features
+
+        """
+        cats = self.categories_
+        feature_names = []
+        if input_features is None:
```

Comment:
if not, please validate that `input_features`'s length corresponds to `self.categories_`

Reply:
I have added this, will make a pr